readthedocs/gold/models.py

```diff
@@ -28,3 +30,9 @@ class GoldUser(models.Model):
 
     def __unicode__(self):
         return 'Gold Level %s for %s' % (self.level, self.user)
+
+    @property
+    def num_supported_projects(self):
+        dollars = int(self.level.split('-')[-1])
+        num_projects = int(math.floor(dollars / 5))
```

Comment:
The `5` is a magic constant here. It should have an explicit name and be pushed to the module scope (or model-class scope) for easier visibility.